amethyst_ui/src/lib.rs

```diff
@@ -1,56 +1,26 @@
 //! Provides components and systems to create an in game user interface.
 
-#![doc(html_logo_url = "https://www.amethyst.rs/assets/amethyst.svg")]
 #![warn(missing_docs, rust_2018_idioms, rust_2018_compatibility)]
+#![doc(html_logo_url = "https://www.amethyst.rs/assets/amethyst.svg")]
```

Comment:
yo, I think you moved this by accident (based on the order mentioned in the first comment)